v19.2/unsplit-at.md

```diff
@@ -0,0 +1,192 @@
+        27 | \275\211\022chicago\000\001  | /Table/53/1/"chicago"  | \275\211\022new york\000\001 | /Table/53/1/"new york" | movr          | users      |            | {1,2,3}  | {}               | 2262-04-11 23:47:16.854776+00:00 |            1
+        28 | \275\211\022new york\000\001 | /Table/53/1/"new york" | \275\211\022seattle\000\001  | /Table/53/1/"seattle"  | movr          | users      |            | {1,2,3}  | {}               | 2262-04-11 23:47:16.854776+00:00 |            1
+        29 | \275\211\022seattle\000\001  | /Table/53/1/"seattle"  | \276                         | /Table/54              | movr          | users      |            | {1,2,3}  | {}               | 2262-04-11 23:47:16.854776+00:00 |            1
+(4 rows)
+~~~
+
+Now unsplit the table:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> ALTER TABLE users UNSPLIT AT VALUES ('chicago'), ('new york'), ('seattle');
+~~~
+~~~
+              key              |         pretty
++------------------------------+------------------------+
+  \275\211\022chicago\000\001  | /Table/53/1/"chicago"
+  \275\211\022new york\000\001 | /Table/53/1/"new york"
+  \275\211\022seattle\000\001  | /Table/53/1/"seattle"
+(3 rows)
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT * FROM crdb_internal.ranges WHERE table_name='users';
+~~~
+~~~
+  range_id |          start_key           |      start_pretty      |           end_key            |       end_pretty       | database_name | table_name | index_name | replicas | learner_replicas | split_enforced_until | lease_holder
++----------+------------------------------+------------------------+------------------------------+------------------------+---------------+------------+------------+----------+------------------+----------------------+--------------+
+        21 | \275                         | /Table/53              | \275\211\022chicago\000\001  | /Table/53/1/"chicago"  | movr          | users      |            | {1,2,3}  | {}               | NULL                 |            1
+        27 | \275\211\022chicago\000\001  | /Table/53/1/"chicago"  | \275\211\022new york\000\001 | /Table/53/1/"new york" | movr          | users      |            | {1,2,3}  | {}               | NULL                 |            3
+        28 | \275\211\022new york\000\001 | /Table/53/1/"new york" | \275\211\022seattle\000\001  | /Table/53/1/"seattle"  | movr          | users      |            | {1,2,3}  | {}               | NULL                 |            2
+        29 | \275\211\022seattle\000\001  | /Table/53/1/"seattle"  | \276                         | /Table/54              | movr          | users      |            | {1,2,3}  | {}               | NULL                 |            1
+(4 rows)
+~~~
+
+The `split_enforced_until` column is now `NULL` for all ranges in the `users` table. The table is still split into ranges at `'chicago'`, `'new york'`, and `'seattle'`, but the split is no longer enforced.
```

Comment:
Might be good to mention that "now the system will be free to re-merge the ranges at its convenience *once you re-enable the `kv.range_merge.queue_enabled` setting to `true`*.  At least according to [the Range Merges doc](https://www.cockroachlabs.com/docs/v19.1/range-merges.html) you will need to have turned off the range merge queue to even be able to SPLIT AT - if I am reading that correctly?  I think we should at least make mention of that in the overview (I'll edit the comment on the overview as well).